docs/analyze.md

```diff
@@ -139,9 +139,7 @@ In TPv2, DataCollectors are loaded from `TestAdaptersPaths` specified in runSett
 
 ## Working with Code Coverage<a name="coverage"></a>
 ### Setup a project
-Add a reference to the `Microsoft.CodeCoverage` [nuget package][coveragenuget] to your project. This will bring in
-coverage infrastructure for a test project. Here's a sample project file, please note the xml entities marked as
-`Required`.
+Here's a sample project file, please note the xml entity marked as `Required`. Previously, the `Microsoft.VisualStudio.CodeCoverage` was required, but is now shipped with the SDK.
```

Comment:
Update Microsoft.Net.Test.Sdk version to 15.5.0 and MSTest.TestAdapter to 1.2.0 , Microsoft.VisualStudio.CodeCoverage was required if Microsoft.Net.Test.Sdk < 15.5.0-preview-20170914-09, See [Dependencies](https://www.nuget.org/packages/Microsoft.NET.Test.Sdk/15.5.0-preview-20170914-09), Update the comment accordingly.